src/handlers.utils.js

```diff
@@ -0,0 +1,56 @@
+const R = require('ramda');
+const SlackTemplate = require('claudia-bot-builder').slackTemplate;
+const { InvalidUserInputError, InvalidCallbackActionError } = require('./handlers.errors');
+
+const validateSendingUser = (userRepository, handler) => async (message, ...extra) => {
```

Comment:
Which variant is most useful, this async one or the sync one below?

Pros (of this one):
* Handler can do other async stuff in parallel.

Cons (of this one):
* More code in handler (need to await promise).
* We need to throw a special exception type for the control flow to work (handled explicitly in `bot.js`).